lib/codecs/src/encoding/format/csv.rs

```diff
@@ -59,16 +84,35 @@ pub struct CsvSerializerOptions {
     pub fields: Vec<ConfigTargetPath>,
 }
 
+impl Default for CsvSerializerOptions {
+    fn default() -> CsvSerializerOptions {
+        CsvSerializerOptions {
+            delimiter: b',',
+            double_quote: true,
+            escape: b'"',
+            fields: vec![]
+        }
+    }
+}
+
 /// Serializer that converts an `Event` to bytes using the CSV format.
 #[derive(Debug, Clone)]
 pub struct CsvSerializer {
+    delimiter: u8,
+    double_quote: bool,
+    escape: u8,
     fields: Vec<ConfigTargetPath>,
```

Comment:
This can be simplified:

```suggestion
#[derive(Debug, Clone, Default)]
pub struct CsvSerializer {
    config: CsvSerializerConfig,
```